app/models/participant_profile/npq.rb

```diff
@@ -5,7 +5,7 @@ class ParticipantProfile::NPQ < ParticipantProfile
   belongs_to :school, optional: true
   belongs_to :npq_course, optional: true
 
-  has_one :validation_data, class_name: "NPQValidationData", foreign_key: :id, dependent: :destroy
+  has_one :validation_data, class_name: "NPQApplication", foreign_key: :id, dependent: :destroy
```

Comment:
```suggestion
  has_one :npq_application, dependent: :destroy
```

and the subsequent calls to `profile.validation_data` => `profile.npq_application`

Reply:
Personally, I'd like to see the actual validation data extracted since that's not really part of the application and the application record pointing to it.